.github/workflows/ci.yml

```diff
@@ -23,6 +23,7 @@ jobs:
           - "++2.13.3 testsJS/test"
           - "download-scala-library ++2.13.3 testsJVM/slow:test"
           - "mima"
+          - "dottytests/test"
```

Comment:
why?

Reply:
I think it will be useful to run it from time to time, but I don't think we need to run it on each PR since most of the cases will always stay the same. I think it would be be nice to be able to run an equivalent of the community build even, but it will be much useful to just run a script when changing the Dotty parser.

Reply:
removed